lib/spree/money.rb

```diff
@@ -28,13 +28,7 @@ def to_s
     end
 
     def to_html(options = { html_wrap: true })
-      output = @money.format(@options.merge(options))
-      if options[:html_wrap]
-        # 1) prevent blank, breaking spaces
-        # 2) prevent escaping of HTML character entities
-        output = output.sub(" ", "&nbsp;").html_safe
-      end
```

Comment:
Great! :fire:

I don't see anywhere that the `options` argument ever gets passed to this method, so we could even remove the argument entirely, but it's not important :ok_hand: